packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@microsoft/fast-element';
+
+import {} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host {
+  }
+
+  :host([dividerAlignContent='start']) {
```

Comment:
We'll want these attribute selectors to get updated according to the notes in the class file.